back/src/domains/inclusion-connected-users/use-cases/CreateUserForAgency.ts

```diff
@@ -40,6 +41,11 @@ export const makeCreateUserForAgency = createTransactionalUseCase<
 
     const user = await getUserIdAndCreateIfMissing(uow, inputParams, deps);
 
+    const isUserAlreadyLinkedToAgency = toPairs(agency.usersRights).some(
+      ([id]) => id === user.id,
+    );
```

Comment:
keys ça suffit dans ce cas :

```suggestion
    const isUserAlreadyLinkedToAgency = keys(agency.usersRights).some(
      (id) => id === user.id,
    );
```